Sources/PointFree/Pricing.swift

```diff
@@ -584,8 +584,15 @@ func redirectActiveSubscribers<A>(
           )
           ?? throwE(unit)
 
-        return (userSubscription.run.parallel <|> ownerSubscription.run.parallel)
-          .sequential
+        let race = (userSubscription.run.parallel <|> ownerSubscription.run.parallel).sequential
+
+        return EitherIO(run: race)
+          .flatMap {
+            $0.stripeSubscriptionStatus == .active
```

Comment:
@mbrandonw Maybe I am misreading, but looks like the pricing page redirect logic was merely checking for a subscription at all, not whether or not it was active.

Reply:
On the road so hard to know for sure, but I believe I did that thinking the user would fix their subscription from the account page. 

Guess I’m a lil murky on when a user should go through the pricing page again

Reply:
Gotcha. OK I think there's more nuance here to handle `pastDue` accounts! Thanks!